common/src/main/java/com/datastax/oss/common/sink/RetryableException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.common.sink;
+
+/** Processing can be retried. */
+public class RetryableException extends RuntimeException {
```

Comment:
what is the purpose of introducing this exception? I don't see any direct usage of that or logic that relies on its type.

Reply:
It is only thrown in case of InterruptedException
Dropping now this class and replacing it with RuntimeException, KafkaConnect and Pulsar IO should treat RuntimeExceptions as processing errors and retry.